apps/web/pages/v2/settings/my-account/profile.tsx

```diff
@@ -166,7 +166,7 @@ const ProfileView = () => {
               <TextField
                 name="username"
                 label={t("personal_cal_url")}
-                addOnLeading="https://"
+                addOnLeading="https://cal.com/"
```

Comment:
```suggestion
                addOnLeading={process.env.NEXT_PUBLIC_WEBSITE_URL}
```

Reply:
Do all self-hosted users use `NEXT_PUBLIC_WEBSITE_URL`?

Reply:
I believe after enviroments migrations this should be the one that everybody should use.

Reply:
or NEXT_PUBLIC_WEBAPP_URL you mean?